dockerfiles/test.Dockerfile

```diff
@@ -0,0 +1,64 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
```

Comment:
for testing only?
what to test ?

Reply:
test.Dockerfile can be used to create a test container,  we can exec into that container and test the comminication between services or something, it;s not necessary, but we can keep